cyber.latex

```diff
@@ -0,0 +1,264 @@
+\documentclass[$if(fontsize)$$fontsize$,$endif$$if(lang)$$babel-lang$,$endif$$if(papersize)$$papersize$paper,$endif$$for(classoption)$$classoption$$sep$,$endfor$]{$documentclass$}
```

Comment:
Because I need to change things that aren't variables. For example, I took out the

```
Chapter 1
General Information
```

and made it 

```
1    General Information
```

.